lang/fr.js

```diff
@@ -49,7 +49,11 @@ export default {
 	"components.meter-mixin.progressIndicator": "Indicateur de progrès",
 	"components.more-less.less": "moins",
 	"components.more-less.more": "plus",
+	"components.overflow-group.moreActions": "More Actions",
```

Comment:
Weird that `fr-fr` got the translation for this but `fr` (which we've mapped from `fr-CA` did not.